doc/GettingStarted.md

```diff
@@ -47,39 +44,4 @@ The generated solution will contain:
 
     ![A screenshot of the generated targets](./Learn/images/GeneratedTargets-min.png)
 
-* Click the "play" button, or press F5 to start debugging. The necessary projects in the solution will be compiled and deployed based on the target platform.
```

Comment:
Same here, the unit tests and UI tests are still available. They're not present by default, which we could explain.

Reply:
@eriklimakc this section will be reviewed with this PR @eriklimakc #2251 (files)?